### ISSUE TITLE
Incompatible with envoy grpc-web filter

### ISSUE THREAD

**Issue by xiaodongw** [improbable-eng/js-browser-headers#14]
I have been testing the improbable-eng/grpc-web client with the envoy grpc-web filter, I got the error "Response closed without grpc-status".

I found envoy sends back the trailer "grpc-status:0", but the code in BrowserHeaders expects there is a space after the colon.

```
appendFromString(str: string): void {
    const pairs = str.split("\r\n");
    for (let i = 0; i < pairs.length; i++) {
      const p = pairs[i];
      const index = p.indexOf(": ");
      if (index > 0) {
        const key = p.substring(0, index);
        const value = p.substring(index + 2);
        this.append(key, value);
      }
    }
  }
```
I think the fix should be 
```
appendFromString(str: string): void {
    const pairs = str.split("\r\n");
    for (let i = 0; i < pairs.length; i++) {
      const p = pairs[i];
      const index = p.indexOf(":");
      if (index > 0) {
        const key = p.substring(0, index);
        const value = p.substring(index + 1).trim();
        this.append(key, value);
      }
    }
  }
```

**Comment by xiaodongw** [comment]
Actually there is already a PR for it https://github.com/improbable-eng/js-browser-headers/pull/13

**Comment by MarcusLongmuir** [comment]
Fixed by #13
